app/models/user.rb

```diff
@@ -1,4 +1,7 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable, :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
Thank you, for the feedback, @Owusu-Desmond.

Looking at the error, it seems like you entered a random text instead of a URL which then throws the error. We will try and handle the case where the photo input is not a valid URL.

Thanks again.